docs/advanced-features/output-file-tracing.md

```diff
@@ -64,7 +64,23 @@ module.exports = {
 }
 ```
 
-- There are some cases in which Next.js might fail to include required files, or might incorrectly include unused files. In those cases, you can export page configs props `unstable_includeFiles` and `unstable_excludeFiles` respectively. Each prop accepts an array of [minimatch globs](https://www.npmjs.com/package/minimatch) relative to the project's root to either include or exclude in the trace.
```

Comment:
why not keep this option as well?

Reply:
We can potentially leave it although doesn't seem ideal to configure in this way per: not being supported for API routes as these files aren't required during build to gather the configs, having to duplicate includes/excludes for multiple pages, and causing more confusion for where the globs were meant to be relative to

Reply:
-- There are some cases in which Next.js might fail to include required files, or might incorrectly include unused files. In those cases, you can export page configs props `unstable_includeFiles` and `unstable_excludeFiles` respectively. Each prop accepts an array of [minimatch globs](https://www.npmjs.com/package/minimatch) relative to the project's root to either include or exclude in the trace.